server/src/main/java/org/elasticsearch/action/admin/indices/get/GetIndexResponse.java

```diff
@@ -69,6 +70,11 @@ public GetIndexResponse(
         }
     }
 
+    /**
+     * The only usage of this constructor is for BwC cross-cluster transforms for clusters before v8.2. The ML team is aware that we
+     * don't need to support that anymore now that we're on v9. Once they remove that BwC code, we can remove this constructor as well.
+     */
+    @UpdateForV10(owner = UpdateForV10.Owner.DATA_MANAGEMENT)
     GetIndexResponse(StreamInput in) throws IOException {
```

Comment:
I reached out to the ML team and they confirmed that the code that uses the GET index action is for BwC to a version that is no longer supported (v8.2) for cross-cluster transforms. This last usage is:
https://github.com/elastic/elasticsearch/blob/b15a4a198a23e562ba7cabe2f79db60cf006cf38/server/src/main/java/org/elasticsearch/action/admin/indices/get/GetIndexAction.java#L19-L19